tests/correctness/constants.rs

```diff
@@ -214,3 +214,69 @@ fn constant_expressions_used_in_array_declaration() {
     //THEN we expect that the array had 10 elements and was filled accordingly
     assert_eq!(main.i, [10, 11, 30, 31, 50, 51, 70, 71, 90, 91]);
 }
+
+#[test]
```

Comment:
I think it's better to add a Codegen test and a parser test here.
The parser test makes sure we could see the LDT#.. correctly and created a date
The codegen test will also make sure we created the right date